backend/app.py

```diff
@@ -56,11 +57,12 @@ def main():
 
 api.add_resource(BannerTask, '/banner', '/banner/<int:id>', endpoint='banner')
 api.add_resource(UsernameTask, '/username')
+api.add_resource(UserTask, '/users')
 
 
 @app.before_request
 def before_request():
-    if app.config.get('is_local'):
+    if os.environ.get('IS_LOCAL') and os.environ.get("TESTING") is None:
```

Comment:
Please document the environment variables IS_LOCAL and TESTING in README.md.

Reply:
I added a testing section to the readme with this information.

Reply:
I don't see anything about IS_LOCAL. Have you replaced `app.config.get('is_local')` with `os.environ.get('IS_LOCAL')` by a mistake or is it intentional?

Reply:
Originally I did it intentionally to keep it consistent with configurations being made through environment variables. On second thought this is out of scope of this feature so I've reverted the change back to `app.config.get('is_local')`